### DISCUSSION TITLE
还有好用的九宫格输入法推荐吗

### DISCUSSION THREAD

**Discussion by Tomp-Akoe** [imfuxiao/Hamster#701] [Q&A]
目前使用的是仓九宫格，整体还可以，但是模糊拼音等细节提升空间还蛮大的，所以想尝试一下其他的九宫格输入方案不知道有没有对应的推荐。

我在github上找到了一个雨燕拼音，但是它好像只能支持安卓，不知道仓可不可以使用。

**Comment by Tomp-Akoe** [comment]
如果没有的话不知道仓九宫格可不可以支持词库更新和模糊拼音优化

**Comment by imfuxiao** [comment]
九宫格输入方案比较特殊，目前是在雾凇词库上增加的针对九宫格输入的配置。

只要是 rime 的方案，都是支持词库更新的，和模糊话拼音设置的，这里不区分是否是九宫格。只是九宫格本身重码就高，在加上模糊音，重码就更高了。

如果你想要自定义方案，可以参考雾凇的九宫格配置。https://github.com/iDvel/rime-ice/blob/main/t9.schema.yaml

**Reply by Tomp-Akoe** [reply]
好嘞！感谢！！昨天刚配置好就果断买会员了哈哈哈哈

**Reply by imfuxiao** [reply]
感谢支持~

**Reply by Tomp-Akoe** [reply]
诶！看了一下配置文件，咱们的仓九宫格就是用的雾凇的库啊！哈哈哈那我没事了
